### PR TITLE
Feature/tickets

### PR DESCRIPTION
**Pull request by SophieTopart**

### CODE REVIEW

File: src/entities/Tickets.ts [hunk 1]
```diff
@@ -27,6 +27,9 @@ class Ticket {
   @Field()
   advancement?: number;
 
+  @Field()
+  project?: string;
```
Comment (FredyLy) [inline src/entities/Tickets.ts:31]: Is it better to name it `projectId` instead of `project`?

File: src/inputs/TicketInput.ts [hunk 2]
```diff
@@ -25,6 +25,9 @@ export default class TicketInput {
   @Field({ nullable: true })
   advancement?: number;
 
+  @Field({ nullable: true })
+  project?: string;
```
Comment (FredyLy) [inline src/inputs/TicketInput.ts:29]: Same here

File: src/inputs/TicketInputUpdate.ts [hunk 3]
```diff
@@ -25,6 +25,9 @@ export default class TicketInputUpdate {
   @Field({ nullable: true })
   advancement?: number;
 
+  @Field({ nullable: true })
+  project?: string;
```
Comment (FredyLy) [inline src/inputs/TicketInputUpdate.ts:29]: Same here

File: src/models/Tickets.ts [hunk 4]
```diff
@@ -9,6 +9,7 @@ const TicketSchema = new Schema({
   initial_time_estimated: Number,
   total_time_spent: Number,
   advancement: Number,
+  project: String,
```
Comment (FredyLy) [inline src/models/Tickets.ts:12]: Same here

File: src/resolvers/TicketsResolver.ts [hunk 5]
```diff
@@ -6,12 +6,21 @@ import TicketInput from '../inputs/TicketInput';
...
 @Resolver()
 class TicketsResolver {
   @Query(() => [Ticket])
   async allTickets() {
     try {
       const getAllTickets = await TicketsModel.find();
+      for (let i = 0; i < getAllTickets.length; i += 1) {
+        getAllTickets[i].advancement = getAdvancement(getAllTickets[i]);
+      }
+      console.log(getAllTickets);
```
Comment (FredyLy) [inline src/resolvers/TicketsResolver.ts:23]: Should be removed

File: src/resolvers/TicketsResolver.ts [hunk 6]
```diff
@@ -22,6 +31,8 @@ class TicketsResolver {
   async getOneTicket(@Arg('id', () => String) ticketId: IdInput) {
     try {
       const getOneTicket = await TicketsModel.findById(ticketId);
+      getOneTicket.advancement = getAdvancement(getOneTicket);
+      console.log(getOneTicket.advancement);
```
Comment (FredyLy) [inline src/resolvers/TicketsResolver.ts:35]: Remove the console log here

**Review by FredyLy — CHANGES_REQUESTED** [review]
Seems really good.
Just some remaining logs and renaming to be reviewed.

**Review by FredyLy — APPROVED** [review]

**Review by dev-talbi — APPROVED** [review]

**Pull request merged**